doc/public-images/source/index.rst

```diff
@@ -66,37 +66,50 @@ without the need for an external IP address (EIP / Floating-IP).
 +----------------------+-------------------------------------+------------------------------+--------------------------------------------------------------------------+
 | 100.125.0.16         | ntp02.eu-nl.otc-service.com         | NTP                          | AZ2                                                                      |
 +----------------------+-------------------------------------+------------------------------+--------------------------------------------------------------------------+
-| 198.19.33.237        | vendordata.eu-de.otc-service.com    | Vendordata OpenStack (HTTP)  | first boot provisioning                                                  |
+| | 198.19.59.175      |                                     |                              |                                                                          |
```

Comment:
why you use double pipes?
Please remove it

Reply:
The double pipes are explicitly wantend to have multiple IPs in one cell.